partiql-tests-data/eval/primitives/operators/bag-operators.ion

```diff
@@ -244,8 +392,8 @@ bagOperators::[
     }
   },
   {
-    name:"OUTER UNION with ORDER BY LIMIT on children and bag op",
-    statement:"(SELECT a, tbl FROM t1 ORDER BY a LIMIT 2) OUTER UNION ALL (SELECT a, tbl FROM t2 ORDER BY a LIMIT 2) ORDER BY a LIMIT 2",
+    name:"SQL UNION with ORDER BY LIMIT on children and bag op",
```

Comment:
Remaining tests clarify that only SQL set ops allow for using `ORDER BY`, `LIMIT`, and `OFFSET`. PartiQL bag ops should not allow for using those other clauses.

Reply:
Based on offline discussion, we will permit using `LIMIT`, `OFFSET`, and `ORDER BY` on the outer bag ops so the earlier comment I left is no longer relevant. I changed those parsing failure tests to parsing success tests.

---

Regarding the modeling of `QuerySet` in PLR, I talked w/ Josh earlier and it was not intentional to allow `LIMIT`, `OFFSET`, and `ORDER BY` for bag ops and general expressions (so `[1, 2, 3] LIMIT 1` is also currently valid). PLR's modeling allowed for a somewhat simpler modeling in the AST and parser. Since the SQL and PartiQL specs do not currently permit or forbid using those clauses on general expressions, perhaps we can say it's implementation-defined for now. We can always disallow those clauses on general expressions in the future.

In my PLK PR, I'll choose to disallow those clauses on general expressions to be a bit more restrictive until it's more specified.